source/Octopus.Manager.Tentacle/Octopus.Manager.Tentacle.csproj

```diff
@@ -10,6 +10,7 @@
     <UseWindowsForms>true</UseWindowsForms>
     <UseWPF>true</UseWPF>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <EnableWindowsTargeting>true</EnableWindowsTargeting>   
```

Comment:
May not be needed for this PR but this allows Linux to build the net6.0 version of these Windows-specific apps
If we don't have these the build fails